### PR TITLE
Feature/icons

### PR DESCRIPTION
**Pull request by james-barnard-consultant**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,11 @@ dist
 
 # TernJS port file
 .tern-port
+packages/icons/dist_unminified/index.cjs.js
```
Comment (randyhoulahan) [inline .gitignore]: remove all these please

File: packages/icons/.vscode/extensions.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (randyhoulahan) [inline packages/icons/.vscode/extensions.json]: remove this file

File: packages/icons/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Vue 3 + Vite
```
Comment (randyhoulahan) [inline packages/icons/README.md]: leave empty for now

File: packages/icons/package.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        "style": "./dist/style.css",
+        "src": "./src",
+        "dist": "./dist",
+        "default": "./dist/umd/index.min.js"
+      },
+      "./dist/legacy/umd/index.umd.min.js"
+    ]
+  },
+  "dependencies": {
+    "vue": "3.2.23",
```
Comment (randyhoulahan) [inline packages/icons/package.json]: "vue": "3.2.31",
    "vue-i18n": "9.2.0-beta.30"

File: packages/icons/package.json [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    "vue-i18n": "9.2.0-beta.23"
+  },
+  "devDependencies": {
+    "@scbd/dist-builder": "^0.0.1-beta",
```
Comment (randyhoulahan) [inline packages/icons/package.json]: not latest

File: packages/icons/package.json [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    "@vitejs/plugin-vue": "1.10.2",
```
Comment (randyhoulahan) [inline packages/icons/package.json]: remove vite related as in dist builder

File: packages/icons/public/index.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    <script async src="https://ga.jspm.io/npm:es-module-shims@1.3.0/dist/es-module-shims.js"></script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="app"></div>
+
+    <script type="importmap">
+      {
+        "imports": {
+          "vue": "https://cdn.cbd.int/vue@3.2.23/dist/vue.esm-browser.prod.js",
```
Comment (randyhoulahan) [inline packages/icons/public/index.html]: match versions from package

File: packages/icons/public/index.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+          "vue-i18n": "https://cdn.cbd.int/vue-i18n@9.2.0-beta.23/dist/vue-i18n.esm-browser.prod.js"
+        }
+      }
+    </script>
+    <script type="module" >
+
+          import { createApp } from "vue";
+          import { createI18n } from "vue-i18n";
+          import App from "./index.es.js";
```
Comment (randyhoulahan) [inline packages/icons/public/index.html]: is this correct file name ... no build should have .es. in it at this point

File: packages/icons/public/preview/import-map.json [hunk 9]
```diff
@@ -0,0 +1,7 @@
+{
+  "imports": {
+    "vue": "https://cdn.cbd.int/vue@3.2.30/dist/vue.esm-browser.prod.js",
```
Comment (randyhoulahan) [inline packages/icons/public/preview/import-map.json]: same as versions from package

File: packages/icons/src/locales/index.js [hunk 10]
```diff
@@ -0,0 +1 @@
...:"cancelar","question-circle":"círculo de preguntas","trash-o":"basura-o","cloud-upload":"carga en la nube","chevron-down":"chevron-down"},"fr":{"close":"Fermer","edit":"Éditer","help":"aider","earth":"Terre","delete":"effacer","file":"déposer","upload to cloud":"télécharger dans le cloud","down":"vers le bas","search":"rechercher","filter":"filtre","infoOutlined":"info-décrit","cancel":"Annuler","question-circle":"question-cercle","trash-o":"poubelle-o","cloud-upload":"téléchargement en nuage","chevron-down":"chevron vers le bas"},"ru":{"close":"Закрыть","edit":"редактировать","help":"помощь","earth":"земля","delete":"Удалить","file":"файл","upload to cloud":"загрузить в облако","down":"вниз","search":"поиск","filter":"фильтр","infoOutlined":"информация обрисована в общих чертах","cancel":"отменить","question-circle":"круг вопросов","trash-o":"мусор","cloud-upload":"облачная загрузка","chevron-down":"шеврон вниз"},"zh":{"close":"关闭","edit":"编辑","help":"帮助","earth":"地球","delete":"删除","file":"文件","upload to cloud":"上传到云端","down":"下","search":"搜索","filter":"筛选","infoOutlined":"信息概述","cancel":"取消","question-circle":"问题圈","trash-o":"垃圾-o","cloud-upload":"云端上传","chevron-down":"V字型向下"}}}
```
Comment (randyhoulahan) [inline packages/icons/src/locales/index.js]: remove unsued

**Comment by james-barnard-consultant** [comment]
All comments are resolved

**Pull request merged**